src/app/Plans/types.ts

```diff
@@ -46,3 +46,13 @@ export interface IMigration {
     StorageReady: boolean;
   };
 }
+
+export interface IVMs {
```

Comment:
As mentioned above, let's name this interface `IVM` since it represents a singular VM. Multiple VMs are represented by `IVM[]`.

Reply:
This one is obsolete, superseded by  https://github.com/konveyor/virt-ui/pull/20/files